libs/api/prisma/migrations/20241126132733_migrate_articles_and_pages_to_leave_revision_history/migration.sql

```diff
@@ -43,7 +43,7 @@ WHERE "publishedAt" IS NULL;
 UPDATE "articles" a
 SET "publishedAt" = ar."publishedAt"
 FROM "articles.revisions" ar
-WHERE ar.id = a."publishedId";
+WHERE ar.id = a."publishedId" AND ar."publishedAt" IS NOT NULL;
```

Comment:
Müsste das nicht in eine neu migration? Da diese sonst nicht mehr ausgeführt wird bei medien bei dennen diese bereits deployed ist?

Reply:
Nur Tsri war betroffen und die 2 Artikel hab ich manuell geflickt. Ist damit es bei der Migration von Bajour/Mannschaft nicht passiert.